src/tools/prte/prte.c

```diff
@@ -1136,17 +1136,18 @@ int main(int argc, char *argv[])
         pname.rank = 0;
     }
     if (PMIX_RANK_INVALID != pname.rank) {
-        PMIX_INFO_CREATE(iptr, 1);
-        PMIX_INFO_LOAD(&iptr[0], PMIX_IOF_PUSH_STDIN, NULL, PMIX_BOOL);
+        pmix_info_t *iptr2;
```

Comment:
Okay, I'm confused - how does changing the name from `iptr` to `iptr2` impact a memory leak? I see no other code change here.

Reply:
Was asking because I've never been a fan of declaring variables in the middle of functions - we've suffered thru a bunch of "shadow" declarations over the years, which is one reason why I turned on the "devel-check" by default for developers. I'd prefer we use the variables declared at the beginning and create/free them whenever they are used.